truffle-config.js

```diff
@@ -45,38 +44,35 @@ module.exports = {
     // tab if you use this network and you must also set the `host`, `port` and `network_id`
     // options below to some value.
     //
-    development: {
-     host: "127.0.0.1",     // Localhost (default: none)
-     port: 8545,            // Standard Ethereum port (default: none)
-     network_id: "*",       // Any network (default: none)
-    },
-
+    // development: {
```

Comment:
Hmmm do we want to keep it commented out? Or can we just remove it completely?

Reply:
I'll remove it